modules/widgets/customizer-utils.js

```diff
@@ -59,6 +59,12 @@ wp.isJetpackWidgetPlaced = function( placement, widgetName ) {
 					} else if ( wp.isJetpackWidgetPlaced( placement, 'eu_cookie_law_widget' ) ) {
 						// Refresh EU Cookie Law
 						placement.container.fadeIn();
+					} else if ( wp.isJetpackWidgetPlaced( placement, 'jetpack_simple_payments_widget' ) ) {
+						// Refresh Simple Payments Widget
+						try {
+							const buttonId = $( '.jetpack-simple-payments-button', placement.container ).attr( 'id' ).replace( '_button', '' );
+							PaypalExpressCheckout.renderButton( null, null, buttonId, null );
+						} catch ( e ) {}
```

Comment:
What sort of error is thrown? Is there nothing we should clean up in this case?

Reply:
Optional, but the `if, else if` blocks look like they can be moved to a switch or early return structure.

Reply:
Re the `try catch`: I've simply copied how the [SP module does that](https://github.com/Automattic/jetpack/blob/6c1971e6bed7d3df793392a7a58ffe0afaeeb5fe/modules/simple-payments/simple-payments.php#L111), as I don't think this PR should be involved in the SP's core functionality.

Re the `if else if`: you're probably right, the early return should work great here. 👍